app/assets/stylesheets/application.scss

```
@@ -0,0 +1,63 @@
+body {
+  padding-top: 70px;
+}
+
+.header_logo {
+  width: 221px;
+  margin-left:20px;
+}
+
+.user_icon {
+  width:40px;
+  height:40px;
+}
+
+.dropdown-menu li a{
+  text-decoration: none;
+  color:black;
+}
+
+.logout {
+  text-align: center;
+  
+}
+
+/*まだサインインしていない場合       */
+.login  {
```

```suggestion
.login {
```
細かいですが、半角スペースは１つの方がいいですね！